inference-engine/src/mkldnn_plugin/mkldnn_graph_optimizer.cpp

```diff
@@ -1153,12 +1110,12 @@ void MKLDNNGraphOptimizer::FuseConvolutionSumAndConvolutionSumActivation(MKLDNNG
 
         auto* binConvNode1 = dynamic_cast<MKLDNNBinaryConvolutionNode *>(parent1.get());
         if (binConvNode1) {
-            isSutableParent1 = isSutableParent1 && binConvNode1->canFuse(graphNode);
+            isSutableParent1 = isSutableParent1 && binConvNode1->canFuseSum(graphNode);
```

Comment:
Can we avoid creating new method (canFuseSum)?

Reply:
it was extracted in lambda because this transformation is single case when we can fuse Special Add and BinaryConvolution
method ```canFuse``` for BinaryConvolution now used to check the possibility fuse SimpleOp in BinaryConvolution